.github/workflows/on-release.yml

```diff
@@ -30,8 +30,8 @@ jobs:
           $repo = "${{ github.repository }}"
           $token = "${{ secrets.GITHUB_TOKEN }}"
 
-          # example: "AdminApp.Web-v2.5.1"
-          # output: "AdminApp.Web"
+          # example: "Admin.Api-v2.5.1"
+          # output: "Admin.Api"
           $packageName = ($release -split "-")[0]
```

Comment:
SHOULD: Similar to above change for "Determining Package Version," we should remove this conditional logic and prioritize simplicity / readability in this step's handling.